controllers/JobController.js

```diff
@@ -164,3 +167,289 @@ exports.addJob = [
     }
   },
 ];
+
+// This function sends interview scheduling mail to cadidate
+exports.scheduleInterview = [
+  auth,
+  body("jobId", "jobId must not be empty.").notEmpty(),
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7787263)
```suggestion
	body("jobId", "jobId must not be empty.").notEmpty(),
```